fpm-Dockerfile-block-2

```diff
@@ -39,4 +39,5 @@ RUN set -ex \
 	} | tee php-fpm.d/zz-docker.conf
 
 EXPOSE 9000
+STOPSIGNAL SIGQUIT
```

Comment:
I have searched and cannot find this officially documented anywhere on https://php.net -- it appears the man page really is the only place it's officially noted. 😓

Can we update this to link to https://github.com/php/php-src/blob/17baa87faddc2550def3ae7314236826bc1b1398/sapi/fpm/php-fpm.8.in#L163 in a code comment for future reference?

ala:

```dockerfile
  ... tee ...conf

# https://github.com/php/php-src/blob/17baa87faddc2550def3ae7314236826bc1b1398/sapi/fpm/php-fpm.8.in#L163
STOPSIGNAL SIGQUIT

EXPOSE 9000
CMD ...
```

Reply:
Great digging tho! The man page seems to be accurate at least!

Reply:
@tianon great digging indeed! I'll add it :+1: